hasher-matcher-actioner/Dockerfile

```diff
@@ -13,4 +13,4 @@ FROM public.ecr.aws/lambda/python:3.8 as prod
 ARG DEPS_PATH
 COPY --from=build $DEPS_PATH $DEPS_PATH
 ENV PYTHONPATH $DEPS_PATH
-COPY . .
+COPY hmalib ./hmalib
```

Comment:
What does the `.` do here?

Reply:
`.` refers to the present working directory both in docker context and in the docker image. I'm kind of certain omitting it would make no difference, though.

Reply:
got it, if it is superfluous (as we both think it is) and not part of some weird best practices, we're probably better off removing it